webapp/Connector/src/view/Learn.js

```diff
@@ -132,6 +132,124 @@ Ext.define('Connector.view.Learn', {
         }
     },
 
+    requestAssayExportCSV : function(cmp, item, data) {
+        console.log("export csv");
+        this.requestAssayExport(false, cmp, item, data);
+    },
+
+    requestAssayExportExcel : function(cmp, item, data) {
+        console.log("export excel");
+        this.requestAssayExport(true, cmp, item, data);
+    },
+
+    requestAssayExport : function(isExcel, cmp, item, data) {
+
+        //for metadata
+        var assayName = data.assay_short_name;
+
+        //for variables and descriptions
+        var variablesSchema = "study";
+        var variablesQuery = data.assay_type;
+        var qviewName = "AssayExportView";
+
+        //for antigen
+        var antigen_query = null;
+        if (data.assay_identifier.includes("BAMA")) {
+            antigen_query = "bamaantigen";
+        }
+        else if (data.assay_identifier.includes("NAB")) {
+            antigen_query = "nabantigen";
+        }
```

Comment:
Consider whether it would be worthwhile to generalize this setup based on new properties that we add to the assay metadata. It might make it easier in the future if they wanted to enable/disable exports in new or existing assays. The argument against this is that it would require them to modify and update their import metadata.

Reply:
Good point! I thought about making it more data driven than hard coded query names; since it would have been slightly more work than anticipated plus there were only two queries, i resorted to this shortcut way for now. 
I like your suggestion of adding it to the assay metadata - I've now included a comment along these lines. I also want to check with Drienna next week if there will be other antigens in the near future to make this change worthwhile.